.github/workflows/test-and-build.yaml

```diff
@@ -17,8 +17,8 @@ jobs:
         uses: ./.github/actions/setup-build
       - name: Unit test
         run: |
-          PATH=$PATH:$(go env GOPATH)/bin make build
-          PATH=$PATH:$(go env GOPATH)/bin make unit-test-no-generate
+          make build
+          make unit-test-no-generate
```

Comment:
Why is setting the go bin path no longer necessary? Is this just coming with the container we use now?

Reply:
we don't use a container, its setup by https://github.com/eksctl-io/eksctl/blob/ec903e328ae1f341cce2e0815d89c8b4a1e867db/.github/actions/setup-build/action.yaml#L8 which automatically adds it to the path